.eslintrc-overrides.json

```diff
@@ -0,0 +1,26 @@
+{
+    "overrides": [
+        {
+            "files": ["*.ts"],
+            "rules": {
+                "no-restricted-imports": [
+                    "error",
+                    {
+                        "paths": [
+                            "rxjs/Rx",
```

Comment:
I would also prohibit importing modules from the root module of `@fundamental-ngx/code` and `@fundamental-ngx/platform` since it may cause circular dependencies during development.

Reply:
done